crates/conversions/src/lib.rs

```diff
@@ -1,26 +1,36 @@
-use cairo_felt::Felt252;
-use starknet::core::types::FieldElement;
-use starknet_api::core::Nonce;
-use starknet_api::{
-    core::{ClassHash, ContractAddress},
-    hash::{StarkFelt, StarkHash},
-};
-
 pub mod class_hash;
 pub mod contract_address;
+pub mod dec_string;
 pub mod felt252;
 pub mod field_element;
 pub mod nonce;
-pub mod short_string;
 pub mod stark_felt;
 
-pub trait StarknetConversions {
-    fn to_felt252(&self) -> Felt252;
-    fn to_field_element(&self) -> FieldElement;
-    fn to_stark_felt(&self) -> StarkFelt;
-    fn to_stark_hash(&self) -> StarkHash; // Alias to StarkFelt
-    fn to_class_hash(&self) -> ClassHash;
-    fn to_contract_address(&self) -> ContractAddress;
-    fn to_short_string(&self) -> String;
-    fn to_nonce(&self) -> Nonce;
+pub trait FromConv<T> {
+    fn from_(value: T) -> Self;
```

Comment:
out of curiousity - why `from_` and not more standard `from` ?

Reply:
We cannot implement external traits for types we don't own

Reply:
https://stackoverflow.com/questions/63093321/why-does-rust-prevent-implementing-an-external-trait-for-an-external-struct